transport/src/main/java/io/netty/channel/DefaultChannelConfig.java

```diff
@@ -430,4 +438,14 @@ public ChannelConfig setMessageSizeEstimator(MessageSizeEstimator estimator) {
         msgSizeEstimator = estimator;
         return this;
     }
+
+    private ChannelConfig setPinEventExecutorPerGroup(boolean pinEventExecutor) {
+        this.pinEventExecutor = pinEventExecutor;
+        return this;
+    }
+
+    private boolean getPinEventExecutorPerGroup() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) A getX() method which returns a boolean should be named isX() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3ABooleanGetMethodName)